src/wallet/feebumper.cpp

```diff
@@ -265,11 +259,8 @@ Result CommitTransaction(CWallet* wallet, const uint256& txid, CMutableTransacti
     wtxBumped.fTimeReceivedIsTxTime = true;
     wtxBumped.fFromMe = true;
     CValidationState state;
-    if (!wallet->CommitTransaction(wtxBumped, reservekey, g_connman.get(), state)) {
-        // NOTE: CommitTransaction never returns false, so this should never happen.
-        errors.push_back(strprintf("The transaction was rejected: %s", state.GetRejectReason()));
-        return Result::WALLET_ERROR;
-    }
+    // NOTE: CommitTransaction never returns false.
+    assert(wallet->CommitTransaction(wtxBumped, reservekey, g_connman.get(), state));
```

Comment:
Could make sense to move it out of the assertion, such that the code continues to run if assertions are not compiled.

Reply:
Agree, done.